api.md

```diff
@@ -571,6 +576,16 @@ Drop all chunks from hypertable `conditions` older than 3 months, including depe
 SELECT drop_chunks(interval '3 months', 'conditions', cascade => TRUE);
 ```
 
+Drop all chunks newer than 3 months:
+```sql
+SELECT drop_chunks(newer_than => interval '3 months');
+```
+
+Drop all chunks newer than 3 months and older than 4 months:
+```sql
+SELECT drop_chunks(older_than => interval '3 months', newer_than => interval '4 months', table_name => 'conditions')
```

Comment:
the parameters do not match your description
older then 4 in description but 3 in sql command

Reply:
Lol yes, @RobAtticus also caught that. Fixing.